src/main/scala/viper/silver/plugin/standard/inline/InlineRewrite.scala

```diff
@@ -23,10 +23,11 @@ trait InlineRewrite {
     )(method.pos, method.info, method.errT)
   }
 
-  def rewriteMethod(method: Method, program: Program, prePredIds: Set[String], postPredIds: Set[String]): Method = {
+  def rewriteMethod(method: Method, prePredIds: Set[String], postPredIds: Set[String]): Method = {
     val rewrittenPres = method.pres.map { removeUnfoldings(_, prePredIds) }
     val rewrittenPosts = method.posts.map { removeUnfoldings(_, postPredIds) }
-    val rewrittenBody = method.body.map { removeFoldUnfolds(_, prePredIds, postPredIds) }
+    val allPredIds = prePredIds ++ postPredIds
```

Comment:
I thought of inlining `prePRedIds ++ postPredIds` at the call site, but I'm not sure I'm a fan of doing that